aiida/work/ports.py

```diff
@@ -31,10 +31,34 @@ def get_description(self):
 
         return description
 
+class WithSerializeFct(object):
```

Comment:
I'm not a big fan of the contraction 'fct', if we keep this I would prefer in order:
1) `WithSerialize`
2) `WithSerializeFunction`
3) `WithSerializeFn`

Reply:
Done (renamed to ``WithSerialize``).